python/sglang/srt/server_args.py

```diff
@@ -432,17 +432,26 @@ def print_deprecated_warning(message: str):
             )
             self.page_size = 128
 
-        if self.attention_backend == "trtllm_mla":
+        if (
+            self.attention_backend == "trtllm_mla"
+            or self.decode_attention_backend == "trtllm_mla"
+        ):
             if not is_sm100_supported():
                 raise ValueError(
                     "TRTLLM MLA backend is only supported on Blackwell GPUs (SM100). Please use a different backend."
                 )
 
+            if self.kv_cache_dtype not in ["fp8_e4m3", "fp16", "auto"]:
```

Comment:
nit: is it bf16 or fp16

Reply:
I removed the fp16/bf16 since it is not supported in the server_args for kv-cache-dtype anyways. Valid options are `fp8_e4m3` for fp8 path and `auto` for BFloat16 path for trtllm-mla.

```
fmhaSm100Kernel_QkvBfloat16OBfloat16HQk576HV512HVPerCta512PagedKvDenseP64VarSeqQ16Kv128PersistentSwapsAbForGen
```
FP8 kernel cubin 
```
fmhaSm100Kernel_QkvE4m3OBfloat16HQk576HV512HVPerCta512PagedKvDenseP64VarSeqQ16Kv128PersistentSwapsAbForGen
```